GNUmakefile

```diff
@@ -606,7 +606,7 @@ source-only: all
 	-$(MAKE) -f GNUmakefile.gcc_plugin
 	$(MAKE) -C utils/libdislocator
 	$(MAKE) -C utils/libtokencap
-	$(MAKE) -C utils/aflpp_driver
+	-$(MAKE) -C utils/aflpp_driver
```

Comment:
For me as makefile noob: what does the `-` do?

Reply:
Then `make` ignores the exit code of this command. This is needed when I run `make tests` for a non supported platform for utilos/aflpp_driver. Otherwise `make tests` would stop here with an error and won't reach the tests, that i am interested in.